backend/routes/chat.py

```diff
@@ -202,6 +202,28 @@ def _handle_react_response(self, turn_response, session_id: str):
 
         try:
             for response in turn_response:
+                # Check if this is an error response from LlamaStack
+                if hasattr(response, "error") and response.error:
+                    error_message = response.error.get("message", str(response.error))
+                    logger.error(f"LlamaStack error response: {error_message}")
+
+                    # Handle specific websearch tool error
+                    # Check for this error message: ValueError: Tool group 'builtin::websearch' not found
+                    if "Tool group 'builtin::websearch' not found" in error_message:
+                        error_message = (
+                            "This assistant is not available right now. It needs internet search tools "
+                            "(such as Tavily) that aren't set up yet. Please try a different assistant "
+                            "or contact your administrator."
+                        )
```

Comment:
It looks like `"Tool group 'builtin::websearch' not found"` is in three places: `_handle_react_response`, `_handle_regular_response`, and `async_iterator_to_iterator()` in chat.py, plus again in llama_stack.py. Maybe create a helper:
```
def map_llamastack_error_to_user_message(raw: str) -> str | None:
    if "Tool group 'builtin::websearch' not found" in raw:
        return ("This assistant is not available right now. It needs internet search tools "
                "(such as Tavily) that aren't set up yet. Please try a different assistant "
                "or contact your administrator.")
    return None
```